data/scripts/update_members.sh

```diff
@@ -0,0 +1,13 @@
+mkdir tmp/
+src="../members.ts"
+new="tmp/members-new.ts"
+extra_code="console.log(JSON.stringify(members));"
+sed 's/export//g' "$src" > "$new"
+sed -i '$d' "$new"
+echo "$extra_code" >> "$new"
+npm install -g typescript
+tsc "$new"
```

Comment:
Don't need this, just use `npm run tsc` instead of `tsc` (the website already lists typescript as a dependency)

Reply:
this didn't work either, but somehow just `tsc` without the former line worked. I added this in the next commit, but make sure it is correct?

Reply:
`tsc` worked because you already had it globally installed on your system. It's a bit unexpected that `npm run tsc` didn't work, but I locally confirmed that `yarn tsc` actually works. Can you change to that?